src/pages/Autolending/DefaultRateDropdown.vue

```diff
@@ -0,0 +1,110 @@
+<template>
+	<div>
+		<h3 class="filter-title">
+			Partner default rate
+		</h3>
+		<kv-dropdown-rounded v-model="defaultRate">
+			<option value="0">
+				All
+			</option>
+			<option value="1">
+				1% or less
+			</option>
+			<option value="2">
+				2% or less
+			</option>
+			<option value="3">
+				3% or less
+			</option>
+			<option value="4">
+				4% or less
+			</option>
+			<option value="5">
+				5% or less
+			</option>
+			<option value="6">
+				6% or less
+			</option>
+			<option value="7">
+				7% or less
+			</option>
+			<option value="8">
+				8% or less
+			</option>
+		</kv-dropdown-rounded>
+	</div>
+</template>
+
+<script>
+import _get from 'lodash/get';
+import gql from 'graphql-tag';
+import KvDropdownRounded from '@/components/Kv/KvDropdownRounded';
+
+export default {
+	inject: ['apollo'],
+	components: {
+		KvDropdownRounded,
+	},
+	data() {
+		return {
+			defaultRate: '0',
+		};
+	},
+	apollo: {
+		query: gql`{
+			autolending @client {
+				currentProfile {
+					loanSearchCriteria {
+						filters {
+							defaultRate {
+								max
+							}
+						}
+					}
+				}
+			}
+		}`,
+		preFetch: true,
+		result({ data }) {
+			// eslint-disable-next-line max-len
+			const defaultRateMax = _get(data, 'autolending.currentProfile.loanSearchCriteria.filters.defaultRate.max');
```

Comment:
You can add `|| 0` to the end of this line to make 0 the default value used when nothing is defined